src/data/quot.lean

```diff
@@ -27,7 +27,7 @@ local notation `⟦`:max a `⟧` := quot.mk _ a
 
 instance [inhabited α] : inhabited (quot ra) := ⟨⟦default _⟧⟩
 
-protected def hrec_on₂ (qa : quot ra) (qb : quot rb) (f : ∀ a b, φ ⟦a⟧ ⟦b⟧)
+protected def hrec_on₂ (qa : quot ra) (qb : quot rb) (f : Π a b, φ ⟦a⟧ ⟦b⟧)
```

Comment:
Do you mind adding a doc string while you're making edits here?

Reply:
I added some docstrings.